quantinuum_schemas/models/backend_config.py

```diff
@@ -224,6 +229,72 @@ class QulacsConfig(BaseBackendConfig):
     result_type: str = "state_vector"
 
 
+class QuantinuumSystemEmulationConfig(BaseModel):
+    """Additional configuration for Selene configurations with quantum."""
+
+    runtime: SimpleRuntime | HeliosRuntime = Field(default=SimpleRuntime())
+    error_mode: SeleneErrorMode = SeleneErrorMode.NONE
+    custom_noise_model: SeleneCustomNoiseModel | None = None
+
+    @model_validator(mode="after")
+    def check_valid_runtime_and_error_mode(self) -> Self:
+        """Validate the runtime and error mode configuration for the Selene emulator."""
+
+        # To be removed once runtimes are fully implemented.
+        if not isinstance(self.runtime, SimpleRuntime):
+            raise ValueError("Only simple runtime is currently supported.")
+
+        # To be removed once error models are fully implemented.
+        if self.error_mode != SeleneErrorMode.NONE:
+            raise ValueError("Only SeleneErrorMode.NONE is currently supported.")
+
+        if (
+            self.custom_noise_model is None
+            and self.error_mode == SeleneErrorMode.CUSTOM
+        ):
+            raise ValueError(
+                "SeleneErrorMode.CUSTOM requires a custom noise model to be provided."
+            )
+
+        if (
+            isinstance(self.runtime, SimpleRuntime)
+            and self.error_mode != SeleneErrorMode.NONE
+        ):
+            raise ValueError("Selene 'simple' runtime requires SeleneErrorMode.NONE.")
+
+        return self
```

Comment:
Are there any other bits of validation that obviously need to happen?

Reply:
The simple runtime doesn't have a requirement that there is no error model, just that there isn't the (to be made) device-specific error model for helios that requires transport information etc. The Depolarizing error model, for instance, would suit it fine

(Whether should be available on nexus is another matter, but it isn't a selene restriction)

Reply:
The expectation is that everything except "Full" should work with the simple runtime I think, unless we allow parameters in "custom" that can affect ion transport. As a lower bound "simple" noise model should work with "simple" runtime.